src/pages/JobDescribtion/components/JobDescriptionItem.jsx

```diff
@@ -0,0 +1,45 @@
+import { useTheme } from "@emotion/react";
+import {
+  ToggleButton,
+  Grid,
+  ToggleButtonGroup,
+  ListItem,
+  ListItemText,
+} from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+function JobDescriptionItem(props) {
+  const [selected, setSelected] = React.useState(() => [0]);
```

Comment:
What kind of default is this?
Is it not the same as [0]? And why is this an array?

Reply:
it is from the mui code example side. the default is that none of the toggle is activated when the site is loaded

Reply:
In the MUI Documentation is this the default value
<img width="496" alt="image" src="https://github.com/SE2-WS23/Skillverse-Web/assets/37250473/e3d40489-dfd1-4f95-84ce-76ca7bc7475b">